src/start_server.ts

```diff
@@ -17,13 +17,16 @@ import * as http from 'spdy';
 import * as pem from 'pem';
 import * as path from 'path';
 import * as send from 'send';
+import * as proxy from 'http-proxy';
 import * as url from 'url';
 import * as fs from 'mz/fs';
 
 import { makeApp } from './make_app';
 
 import findPort = require('find-port');
 import opn = require('opn');
+// Cant be an import since httpProxy is not a function, but an object instead
```

Comment:
no need to comment on this. Or, if you do want to add a comment, add it above these three since it's the reason all 3 are using require.